classes/projects-define.php

```diff
@@ -59,6 +59,9 @@ public function define_post_type() {
 					'api_key'          => new \Fieldmanager_TextField( array(
 						'label'          => __( 'API Key', 'project-dashboard' ),
 						'serialize_data' => false,
+						'attributes' => array( 'readonly' => true, 'disabled' => true    ),
+						'default_value' => wp_generate_password( 12 ),
```

Comment:
This is going to include some non url friendly characters, looks like `wp_generate_password( 8, false );` will be a better choice.